ras/sosreport.py

```diff
@@ -167,10 +174,8 @@ def test_others(self):
             self.is_fail += 1
             self.log.info("--profile option failed")
 
-        dir_name = self.run_cmd_out("sosreport --batch --tmp-dir=%s --build | "
-                                    "grep located | "
-                                    "cut -d':' -f2" % directory_name).strip()
-        if not os.path.isdir(dir_name):
+        dir_name = self.run_cmd_search("sosreport --batch --tmp-dir=%s --build" % directory_name, directory_name)
+        if not os.path.isdir(dir_name[0].strip()):
```

Comment:
if all the return values of `run_cmd_search` needs a `[0].strip()` can you also move that into the function and name it accordingly?